docs/getting-started-guides/scratch.md

```diff
@@ -242,12 +242,12 @@ documentation](/docs/admin/authentication/#creating-certificates/).
 You will end up with the following files (we will use these variables later on)
 
 - `CA_CERT`
-  - put in on node where apiserver runs, in e.g. `/srv/kubernetes/ca.crt`.
+  - put in on node where apiserver runs, for example, in `/srv/kubernetes/ca.crt`.
```

Comment:
This is a good update to remove the latin abbreviation but the second comma isn't needed - 

    put in on node where apiserver runs, for example, in `/sr

would be better as 

    put in on node where apiserver runs, for example in `/sr